target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vgv952cjw33-e-ir-smp.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "vr9_arcadyan_vgv952cjw33-e-ir.dtsi"
+
+/ {
+	compatible = "arcadyan,vgv952cjw33-e-ir-smp", "lantiq,xway", "lantiq,vr9";
+	model = "Easybox 904xDSL";
```

Comment:
To start with i only looked at this one. I still did not understand what that the VPE variant is all about.

I wonder if we should start with one variant and possibly later add another one. Since i do not know what VPE does and i like the second core, i would suggest this one.

Reply:
VR9 SoC got two MIPS cores. Either you can use one with Linux and one with a proprietary RTOS for making use of the analog telephony (and DECT, ...) stuff (aka. VPE "**V**oice **P**rocessing **E**ngine"), or you can use both with Linux (aka. SMP "**S**ymmertic **M**ulti**p**rocessing") if you don't plan to use the telephony interfaces.